src/js/components/NewTodo.js

```diff
@@ -35,20 +34,30 @@ export default class NewTodo extends Component {
     render() {
         if (!this.state.isNewTodoFormDisplayed) {
             return (
-                <div class="col-sm-offset-5">
-                    <button onClick={this.toggleNewTodoForm.bind(this)}>Add new todo</button>
+                <div className="row">
+                    <div className="col-xs-offset-4 col-sm-offset-5 additional-margin-top--less">
+                        <div className="row">
+                            <button className="btn btn-primary col-xs-5 col-lg-2" onClick={this.toggleNewTodoForm.bind(this)}>Add new todo</button>
+                        </div>
```

Comment:
Avoid using `.bind(this)`. That convension is massy. Much better is binding in constructor ( it would be `this. toggleNewTodoForm =  this. toggleNewTodoForm.bind(this) ` ) or here You can do something like this: `() => this.toggleNewTodoForm()`